fava_investor/modules/performance/balances.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+from typing import Dict, List, Union
+
+from beancount.core import realization
+from beancount.core.data import iter_entry_dates
+from fava.core import FavaLedger
+from util import pairwise
+from util.date import Interval
+
+ConfigDict = Dict[str, Union[str, List[str]]]
+
+
+def filter_nested_accounts_dict(accounts_dict: Dict[str, Dict],
+                                accounts_to_keep: List[str], prev:str = "") -> dict:
```

Comment:
`: List[str]` Type hints are a new feature in python and are not enforced. IMHO, it reduces readability. I'd prefer to not use them.